examples/digital-identity/controllers/share.session.controller.js

```diff
@@ -59,6 +64,16 @@ module.exports = (req, res) => {
         status, code, reason, message,
       };
       console.error(errorData);
-      res.json(errorData);
+      res.end(jsonToFormattedString(errorData));
     });
-};
+});
+
+router.get('/fetchSession/:sessionId', (req, res) => {
```

Comment:
Not really keen on that `fecthSession`...

Maybe we should have a chat about what we want to achieve for the example @dhivya-pattabhiraman ?

Reply:
So, what i had in mind was that when we have web share ready, we will create an example like we have for the Profile example that uses web share for integration (which will under the hood take care of QR code creation and refresh). And we only need to show Session creation and receipt retrieval. Fetching the session will be needed for getting the receipt id. What do you guys think?

Reply:
Hum, I am not sure we'll have web-share ready there by then, also, using web-share would remove the need to use a few of the endpoint...
So, let's do something dry first...
Then, in the same example, once we have web-share, we can put in place another route that would illustrate how to use both of them.

Reply:
We could, or we could expose endpoints as dissociated...